lib/utils/parseRequest.js

```diff
@@ -1,3 +1,6 @@
 module.exports = rawRequest => {
-
+  const parsedRequest = rawRequest.split('\r\n');
+  const [method, path] = (parsedRequest[0].split(' '));
```

Comment:
Note that the parenthesis around `parsedRequest[0].split(' ')` are optional:

```suggestion
  const [method, path] = parsedRequest[0].split(' ');
```

Reply:
Well noted!